helm-charts/riff/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: riff is for functions - a FaaS for Kubernetes
 name: riff
-version: 0.0.0
+version: 0.0.6-snapshot
```

Comment:
Do we need a non-0.0.0 version?

If so, can we use a version that doesn't need to be manually maintained and instead is replaced within the helm-publish script?

Reply:
Oops, I do that when testing locally, I guess we could use 9.9.9 or something similar